### PR TITLE
NG-1284 senior app crashes after long videocall

### PR DESCRIPTION
**Pull request by f1shear**

### CODE REVIEW

**Review by javipascual — APPROVED** [review]

**Review by locomocoroco — APPROVED** [review]

File: android/src/main/java/com/media4care/twilio/video/TwilioVideoPlugin.java [hunk 1]
```diff
@@ -24,20 +24,30 @@ public void load() {
...
+            Intent intent = new Intent(getContext(), TwilioVideoActivity.class);
+            intent.putExtra("roomName", call.getString("roomName"));
+            intent.putExtra("accessToken", call.getString("accessToken"));
+            intent.putExtra("options", call.getObject("options").toString());
 
-        getContext().startActivity(intent);
-        call.success();
+            getContext().startActivity(intent);
+            call.success();
+        } catch (Exception e) {
```
Comment (euclides-jr) [inline android/src/main/java/com/media4care/twilio/video/TwilioVideoPlugin.java]: Android manage activities independently any exception thrown in the started activity will not be caught here: https://stackoverflow.com/questions/21138680/catch-exception-from-child-activity-in-parent-activity

File: android/src/main/java/com/media4care/twilio/video/TwilioVideoPlugin.java [hunk 2]
```diff
@@ -24,20 +24,30 @@ public void load() {
...
     public void leaveTwilioRoom(PluginCall call) {
-        Intent intent = new Intent(TwilioVideoActivity.CLOSE_EVENT);
-        LocalBroadcastManager.getInstance(getContext()).sendBroadcast(intent);
-        call.success();
+        try {
+            Intent intent = new Intent(TwilioVideoActivity.CLOSE_EVENT);
+            LocalBroadcastManager.getInstance(getContext()).sendBroadcast(intent);
+            call.success();
+        } catch (Exception e) {
+            e.printStackTrace();
```
Comment (euclides-jr) [inline android/src/main/java/com/media4care/twilio/video/TwilioVideoPlugin.java]: Same as previous comment any exception in the intent will not be caught here

File: android/src/main/java/com/media4care/twilio/video/TwilioVideoPlugin.java (hunk 1 above)
Comment (f1shear) [reply android/src/main/java/com/media4care/twilio/video/TwilioVideoPlugin.java]: removed

File: android/src/main/java/com/media4care/twilio/video/TwilioVideoPlugin.java (hunk 2 above)
Comment (f1shear) [reply android/src/main/java/com/media4care/twilio/video/TwilioVideoPlugin.java]: removed

**Review by euclides-jr — APPROVED** [review]

**Pull request merged**